wolfcrypt/src/ecc.c

```diff
@@ -6219,6 +6227,14 @@ int wc_ecc_free(ecc_key* key)
     }
 #endif
 
+#ifdef WOLFSSL_SE050
+    //printf("ran se050 key_object_free\n");
```

Comment:
Remember to remove debug statements and replace all comments with `/* comment */` style.